server/src/services/cleanUserInput.js

```diff
@@ -0,0 +1,10 @@
+const cleanUserInput = formInput => {
+    Object.keys(formInput).forEach(field => {
+      if(formInput[field] === "") {
+        delete formInput[field]
+      }
+    })
+    return formInput
+  }
+  
+  export default cleanUserInput
```

Comment:
files should end with a single empty line
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--newline-at-end